modules/network/vpc/README.md

```diff
@@ -1,20 +1,28 @@
 ## Description
 
-This module creates a new VPC network along with a
-[cloud NAT](https://github.com/terraform-google-modules/terraform-google-cloud-nat),
-[Router](https://github.com/terraform-google-modules/terraform-google-cloud-router)
-and common [firewall rules](https://github.com/terraform-google-modules/terraform-google-network/tree/master/modules/firewall-rules).
-This module is based on submodules defined by the
-[Cloud Foundation Toolkit](https://cloud.google.com/foundation-toolkit).
-
-The created cloud NAT (Network Address Translation) allows virtual machines
-without external IP addresses to create outbound connections to the internet.
```

Comment:
nit: "without external IP addresses" was informative (at least to me) and was  not carried over. Considering adding it to the updated version.

Reply:
Addressed nit on HEAD of PR